conftest.py

```diff
@@ -26,26 +28,41 @@
 
 selenium.webdriver.common.utils.is_connectable = _selenium_is_connectable
 
-try:
-    import pytest
 
-    def pytest_addoption(parser):
-        group = parser.getgroup("general")
-        group.addoption(
-            "--build-dir",
-            action="store",
-            default=BUILD_PATH,
-            help="Path to the build directory",
-        )
-        group.addoption(
-            "--run-xfail",
-            action="store_true",
-            help="If provided, tests marked as xfail will be run",
-        )
+def pytest_addoption(parser):
+    group = parser.getgroup("general")
+    group.addoption(
+        "--build-dir",
+        action="store",
+        default=BUILD_PATH,
+        help="Path to the build directory",
+    )
+    group.addoption(
+        "--run-xfail",
+        action="store_true",
+        help="If provided, tests marked as xfail will be run",
+    )
+
+
+def pytest_configure(config):
+    """Monkey patch the function cwd_relative_nodeid returns the description
+    of a test for the short summary table. Monkey patch it to reduce the verbosity of the test names in the table.
+    This leaves enough room to see the information about the test failure in the summary.
+    """
+    old_cwd_relative_nodeid = config.cwd_relative_nodeid
 
+    def cwd_relative_nodeid(*args):
+        result = old_cwd_relative_nodeid(*args)
+        result = result.replace("src/tests/", "")
+        result = result.replace("packages/", "")
+        result = result.replace("::test_", "::")
+        result = result.replace("[chrome]", "")
+        result = result.replace("[firefox]", "")
```

Comment:
When you run locally it is useful to see if it runs for chrome or firefox

Reply:
How about only stripping the browser names if run with `-k chrome` or `-k firefox`?

Reply:
I think too much magic can be confusing for users who are used to normal pytest behavior.